src/utils.rs

```diff
@@ -1,7 +1,7 @@
 // Copyright 2020 TiKV Project Authors. Licensed under Apache-2.0.
 
 #[inline]
-pub(crate) fn real_time_ns() -> u64 {
+pub fn real_time_ns() -> u64 {
```

Comment:
Why `pub`?

Reply:
It will not be exported, because the mod it‘s in is private.